exercises/robot-simulator/src/main/java/GridPosition.java

```diff
@@ -0,0 +1,16 @@
+public final class GridPosition {
+
+    final int x;
+
+    final int y;
+
+    GridPosition(final int x, final int y) {
+        this.x = x;
+        this.y = y;
+    }
+
+    boolean equals(final GridPosition gridPosition) {
```

Comment:
Excellent point.  Agreed.

We also want to balance that with the principle that Exercism code should be exemplary.  

What do you think of some succinct comment above or within `equals()` that include a URL to an accessible description of the rationale behind keeping `equals()` and `hashCode()` in sync, semantically?